modules/base/tool.go

```diff
@@ -45,8 +45,8 @@ func EncodeSha1(str string) string {
 }
 
 func ShortSha(sha1 string) string {
-	if len(sha1) == 40 {
```

Comment:
Yes, please do the modification, we still want to show 10-char on the UI. Thought people could just use 7-char in other cases.